docs/resources/api_implementation.md

```diff
@@ -16,9 +16,16 @@ APIImplementation Resource
 resource "konnect_api_implementation" "my_apiimplementation" {
   provider = konnect-beta
   api_id = "9f5061ce-78f6-4452-9108-ad7c02821fd5"
-  service = {
-    control_plane_id = "9f5061ce-78f6-4452-9108-ad7c02821fd5"
-    id               = "7710d5c4-d902-410b-992f-18b814155b53"
+  api_implementation_control_plane_entity = {
+    control_plane = {
+      control_plane_id = "7710d5c4-d902-410b-992f-18b814155b53"
+    }
+  }
+  api_implementation_gateway_service_entity = {
+    service = {
+      control_plane_id = "9f5061ce-78f6-4452-9108-ad7c02821fd5"
+      id               = "7710d5c4-d902-410b-992f-18b814155b53"
+    }
   }
```

Comment:
This DX isn't great. Can we use `title` via an overlay to rename the block to something like `link_to_service`?

Reply:
Agree about DX, will rename. Couple of questions - 
1. I would imagine we don't want to do this too many times (renaming via overlay) - correct?
2. Would this be something to flag during the initial API design review before it is built?

Reply:
I'll start a conversation with Portal team today

Reply:
I took another look, this seems to be due to [use of oneOf](https://github.com/Kong/platform-api/blob/main/src/konnect/definitions/api-builder/v3/openapi.yaml#L2819), and can be renamed. I'll try using x-speakeasy-name-override and send a PR on platform-api.